src/daikinapi.ts

```diff
@@ -29,12 +30,29 @@ export declare const enum TargetHeatingCoolingState {
 
 export type LogMessage = (level: LoggerLevel, message: string, ...parameters: any[]) => void
 
+export type DataChanged = () => void
+
+// After sending an update to the Daikin API it will return old data for up to 15 seconds, so we
+// delay fetching data after an update by this amount. https://daikinone.com/openapi/documentation/
+export const DAIKIN_DEVICE_WRITE_DELAY_MS = 15*1000;
+
+// User is not interacting with a HomeKit controller - background updates for automations
+export const DAIKIN_DEVICE_BACKGROUND_REFRESH_MS = 180*1000;
+
+// User is interacting with a HomeKit controller - latest data needed
+export const DAIKIN_DEVICE_FOREGROUND_REFRESH_MS = 10*1000;
```

Comment:
Since from all we can see the Daikin iOS app polls this API once every 10 seconds I found it to be a reasonable maximum frequency for when a user is viewing/using the data, i.e. we are at least not worse.